src/main.c

```diff
@@ -931,6 +946,18 @@ static int listener_setup_ssl(h2o_configurator_command_t *cmd, h2o_configurator_
     ssl_ctx = SSL_CTX_new(SSLv23_server_method());
```

Comment:
`SSLv23_server_method()` has been deprecated.
Would it be possible to use `TLS_server_method()` or even stricter one TLSv1_2_server_method().

Reply:
Actually not sure if that matters for picotls.

Reply:
Thank you for your feedback!